ProgramParsing/AHS/MajorParser2021_2022.py

```diff
@@ -38,10 +38,16 @@ def _get_relatedMajor(self, program):
     def __get_courses(self, line):
         # Get courses from line
         line = line.replace("*", "").replace("/", " or ")
+
         coursesA = set(re.findall(r"[A-Z]{2,10}(?:\s[1-9][0-9][0-9][A-Z]?)?(?:\sor\s[A-Z]{2,10}(?:\s[1-9][0-9][0-9][A-Z]?)?){0,10}", line))
         coursesB = set(re.findall(r"(?:[A-Z]{2,10}\s)?[1-9][0-9][0-9][A-Z]?(?:\sor\s[A-Z]{2,10}(?:\s[1-9][0-9][0-9][A-Z]?)?){0,10}", line))
         courses = coursesA.union(coursesB)
         courses = list(courses)
+
+        if ("One" in line and "from the following list" in line) or \
+                "One of" in line or "Two of" in line:
```

Comment:
use regex here: 

if ("One" in line and "from the following list" in line) or re.findall(r"(One|Two|Three|Four|Five|Six|Seven|Eight|Nine|Ten|All) of", line):

Reply:
also, if this is two+ of , are you sure it is [" or ".join(courses)] not [", ".join(courses)]. Do we even have or courses in the DB

Reply:
for the second part with the "or" stuff, theres some weird stuff going on in that Parser, so when we use or, it gets converted to 1 entry into the database, whereas if we use a comma, it enters them as separate entries

Reply:
I think I know what you mean now